.github/workflows/publish-master-npm.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish NPM Package `master`
+on:
+  push:
+    branches:
+      - master
+jobs:
+  publish:
+    name: Publish NPM Package `master`
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+    steps:
+      - name: Checkout Release
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Using Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Generate NPM package
+        run: |
+           make build-npm
+
+      - name: publish new release
+        if: contains(github.ref, 'refs/tags/v')
+        run: |
+           make publish-npm
```

Comment:
`make publish-npm`  needs to generate different package.json version (for example `x.y.z-master.<git-sha>`) when `NPM_RELEASE_TAG=master`. This can be done in `build/set_npm_version.ts`.